ProcessLib/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ APPEND_SOURCE_FILES(SOURCES TES)
 add_subdirectory(HeatConduction)
 APPEND_SOURCE_FILES(SOURCES HeatConduction)
 
+add_subdirectory(RichardsFlow)
+APPEND_SOURCE_FILES(SOURCES RichardsFlow)
```

Comment:
This line is sufficent. Remove the `add_subdirectory(Richards)` above and the empty CMakeLists.txt.